docs/logs/customizing-the-sdk/README.md

```diff
@@ -26,7 +26,9 @@ See [Program.cs](Program.cs) for an example.
 
 ### IncludeFormattedMessage
 
-TODO
+`IncludeFormattedMessage` will invoke a formatter with the signature
+`Func<TState, Exception, string>` to set the value of `LogRecord.FormattedMessage`.
+The default message formatter will only return the value of the `object TState`.
```

Comment:
this is implementation detail. There is no "default" formatter. I'd suggest removing this.

Reply:
My suggestion would be:
`IncludeFormattedMessage` will populate `LogRecord.FormattedMessage` by invoking the formatter from [https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.ilogger.log?view=dotnet-plat-ext-6.0]. Period.

Reply:
High order bits:
1. What `IncludeFormattedMessage` does https://github.com/open-telemetry/opentelemetry-dotnet/blob/2a97920ff0a603837a1121204824955bba739c57/src/OpenTelemetry/Logs/OpenTelemetryLoggerOptions.cs#L36-L37
2. What's the default value.

Low order bits:
1. Enabling it will have extra perf/memory cost (probably no need to mention).